bayesian/api/api_v2.py

```diff
@@ -61,6 +61,32 @@
 metrics = GunicornPrometheusMetrics(api_v2, group_by="endpoint", defaults_prefix=NO_PREFIX)
 
 
+@api_v2.route('/get-token', methods=['GET'])
+@login_required
+@validate_user
+def get_token():
+    """Return 3Scale tokens based on organization"""
+    try:
+        organization = request.args.get('org').lower()
+        if organization and organization.lower() == "crda":
```

Comment:
Why would the organization send crda?
should they not send something like quay,clair or xyz user.
so we can identify who they are?
@mathur07 @yzainee